modules/form-field/react/stories/stories_Checkbox.tsx

```diff
@@ -118,56 +119,64 @@ storiesOf('Components|Inputs/Checkbox/React/Left Label', module)
+          ],
+        },
+        props => {
+          if (props.indeterminate && !props.checked) {
+            return false;
+          }
+          return true;
+        }
+      )}
+      columnProps={permutateProps(
+        {
+          className: [
+            {label: 'Default', value: ''},
+            {label: 'Hover', value: 'hover'},
+            {label: 'Focus', value: 'focus'},
+            {label: 'Focus Hover', value: 'focus hover'},
+            {label: 'Active', value: 'active'},
+            {label: 'Active Hover', value: 'active hover'},
+          ],
+          disabled: [{label: '', value: false}, {label: 'Disabled', value: true}],
+        },
+        props => {
+          if (props.disabled && !['', 'hover'].includes(props.className)) {
+            return false;
+          }
+          return true;
+        }
+      )}
+    >
+      {props => (
+        <Checkbox
+          {...props}
+          onChange={() => {}} // eslint-disable-line no-empty-function
+          label="Checkbox"
+        />
+      )}
+    </ComponentStatesTable>
+  </StaticStates>
+);
+
 storiesOf('Components|Inputs/Checkbox/React/Visual Testing', module)
   .addParameters({component: Checkbox})
   .addDecorator(withReadme(README))
-  .add('States', () => (
-    <StaticStates>
-      <ComponentStatesTable
-        rowProps={permutateProps(
-          {
-            checked: [{value: true, label: 'Checked'}, {value: false, label: 'Unchecked'}],
-            indeterminate: [{value: true, label: 'Indeterminate'}, {value: false, label: ''}],
-            error: [
-              {value: undefined, label: ''},
-              {value: Checkbox.ErrorType.Alert, label: 'Alert'},
-              {value: Checkbox.ErrorType.Error, label: 'Error'},
-            ],
-          },
-          props => {
-            if (props.indeterminate && !props.checked) {
-              return false;
-            }
-            return true;
-          }
-        )}
-        columnProps={permutateProps(
-          {
-            className: [
-              {label: 'Default', value: ''},
-              {label: 'Hover', value: 'hover'},
-              {label: 'Focus', value: 'focus'},
-              {label: 'Focus Hover', value: 'focus hover'},
-              {label: 'Active', value: 'active'},
-              {label: 'Active Hover', value: 'active hover'},
-            ],
-            disabled: [{label: '', value: false}, {label: 'Disabled', value: true}],
-          },
-          props => {
-            if (props.disabled && !['', 'hover'].includes(props.className)) {
-              return false;
-            }
-            return true;
-          }
-        )}
-      >
-        {props => (
-          <Checkbox
-            {...props}
-            onChange={() => {}} // eslint-disable-line no-empty-function
-            label="Checkbox"
-          />
-        )}
-      </ComponentStatesTable>
-    </StaticStates>
-  ));
+  .add('States', () => <CheckboxStates />)
```

Comment:
Story source won't be very useful for these stories, but that is _probably_ okay because we don't expect people to use the source code for these visual stories.

Reply:
yep yep, I think we pulled this out into a variable because `StaticStates` won't make much sense to someone viewing the source code.